### PR TITLE
[SYCL][NFC] Enable group and sub_group tests on CUDA

### PR DESCRIPTION
**Pull request by alexbatashev**

### CODE REVIEW

**Comment by vladimirlaz** [comment]
Could you please create an issue for test which are still disabled?
Also it is better to mark them XFAIL instead of disabling

**Comment by alexbatashev** [comment]
> Could you please create an issue for tests which are still disabled?
> Also it is better to mark them XFAIL instead of disabling

Trackers:
https://github.com/intel/llvm/issues/2252
https://github.com/intel/llvm/issues/2754

File: sycl/test/on-device/sub_group/load_store.cpp [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-// UNSUPPORTED: cuda || cpu
+// XFAIL: cuda, cpu
```
Comment (vladimirlaz) [inline sycl/test/on-device/sub_group/load_store.cpp:2]: It is better to have UNSUPPORTED: on cpu
The problem is seen only for some ISAs. See https://github.com/intel/llvm/issues/2252
```suggestion
// UNSUPPORTED: cpu
// XFAIL: cuda
```

**Review by vladimirlaz — CHANGES_REQUESTED** [review]

**Review by vladimirlaz — APPROVED** [review]

**Comment by vladimirlaz** [comment]
@alexbatashev  could you please propagate the change to https://github.com/intel/llvm-test-suite/tree/intel/SYCL/GroupAlgorithm?

**Pull request merged**